fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/TagUtil.java

```diff
@@ -35,6 +35,9 @@
  * For non-dynamic registry entries, the simpler {@link #isIn(TagKey, Object)} can be used.
  */
 public final class TagUtil {
+	public static String C_TAG_NAMESPACE = "c";
+	public static String FABRIC_TAG_NAMESPACE = "fabric";
```

Comment:
Are these used (search is kinda falling appart on this pr :D )? They seem somewhat redundant imo?

Reply:
Yep. Part of the TagRegistration's two register methods. I kept the registerFabric method in case Fabric adds a new Fabric namespace only tag in future but rn, registerFabric is unused. So the fabric field is only used in the ConventionLogWarnings class at the moment.
![image](https://github.com/FabricMC/fabric/assets/40846040/d33dfcbb-87c4-4791-b045-47fcb3b6a922)

![image](https://github.com/FabricMC/fabric/assets/40846040/1c14d100-e0b9-409d-a7ba-9eac82bd4f2d)

Reply:
Ah ok nice, there is no reason not to have these so its fine 👍